Formula/ipython@5.rb

```diff
@@ -99,7 +145,16 @@ class IpythonAT5 < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec)
+
+    %w[appnope backports.shutil_get_terminal_size decorator enum34 ipython_genutils pathlib2 pexpect pickleshare prompt_toolkit ptyprocess Pygments scandir simplegeneric six traitlets wcwidth].each do |r|
```

Comment:
@tdsmith See ipython/ipykernel#250; basically IPyKernel's `setup.py` expects IPython to be available at installation time so it can call some of its methods for registration. We could comment out some of those calls, but according to that issue, it might not be able to find IPython afterward.